source/includes/_tipser-react-elements.md

```diff
@@ -15,7 +15,7 @@ The code of that page is available as a GitHub [Tipser Elements Bootstrap projec
 
 ***
 
-## Quick Start
+## Quick React Start
```

Comment:
This section header makes no sense for me ;-)

Reply:
Yeah - I think we can't think of something really good here. Shouldn't we consider separate html pages per main section ? One for tipser-elements, another for tipser-sdk. That would help us avoid such situation.